ProjectSidewalk/SidewalkWebpage
# Fixes dev environment setup bug after sunsetting of bintray

Resolves #2662 

Bintray was sunsetted a couple months ago, which means that our first-time dev environment setup was no longer working. We just needed to change the location of the repository we were pulling from in the Dockerfile. While I was at it, I upgraded from node 10 to node 14, because they are no longer providing security updates for node 10. This did not seem to cause any issues, and I believe that we are only using node for our dev environment.

##### Testing instructions
1. Make sure that `make dev` is _not_ running
2. Remove the docker image for `projectsidewalk/web` using `docker rmi <image_id>` (you can leave the database container intact)
3. Prune images and volumes with `docker image prune` and `docker volume prune`
4. Check out this branch and run `make dev`; the container should build and the website should work!

##### Things to check before submitting the PR
